Moderation-Policy.md

```diff
@@ -294,6 +292,8 @@ remove resigning team member from respective permissions and private access.
 **Alexander Tesfamichael** &lt;alex.tesfamichael@gmail.com&gt;
 * [aredridel](https://github.com/aredridel) -
 **Aria Stewart** &lt;aredridel@dinhe.net&gt;
+* [bnb](https://github.com/bnb) -
```

Comment:
This addition has been made to the `### Admins for Node.js Slack community` section, is that intentional? Looking at https://github.com/nodejs/admin/pull/254, it seems that the only change should be the removal of lines 276 and 277 that you have done already at the top.

Reply:
my bad, I just saw the list and assumed it was emeritus without actually reading.